Examples/Run/Reconstruction/Common/SeedingExample.cpp

```diff
@@ -118,23 +118,22 @@ int runSeedingExample(int argc, char* argv[],
   seedingCfg.inputSpacePoints = {
       spCfg.outputSpacePoints,
   };
+
+  Acts::RegionalParameters<SimSpacePoint> regionalParameters;
+  Acts::SeedFilterConfig filterConfig;
+
   seedingCfg.outputSeeds = "seeds";
   seedingCfg.outputProtoTracks = "prototracks";
   seedingCfg.gridConfig.rMax = 200._mm;
   seedingCfg.seedFinderConfig.rMax = seedingCfg.gridConfig.rMax;
 
-  seedingCfg.seedFilterConfig.deltaRMin = 1_mm;
-  seedingCfg.seedFinderConfig.deltaRMin = seedingCfg.seedFilterConfig.deltaRMin;
-  seedingCfg.seedFinderConfig.deltaRMinTopSP =
-      seedingCfg.seedFilterConfig.deltaRMin;
-  seedingCfg.seedFinderConfig.deltaRMinBottomSP =
-      seedingCfg.seedFilterConfig.deltaRMin;
+  filterConfig.deltaRMin = 1_mm;
```

Comment:
isn't this parameter removed?

Reply:
nvm, found my mistake. Not sure if it's actually reasonable to not distinguish between bottom and top in the SeedFilter if we do in the SeedFinder

Reply:
I removed it from the finder since it wasn't used anymore. But this is the filter config